### ISSUE TITLE
Tile controls can't be used with keyboard

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#214]
Controls aren't using a focusable control. Update markup so the i element is focusable or use a different focusable element. Element should also allow someone to use control with keyboard. This will allow someone to open/close and access settings. Moving tiles will be separate issue.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.5.6](https://github.com/blackbaud/skyux/releases/tag/v1.5.6)
